<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEditWindow</class>
 <widget class="QWidget" name="AddEditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="central_frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="shadow_layout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QFrame" name="background_frame">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 220, 215)</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QFrame" name="top_frame">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(170, 0, 0);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="title_label">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>14</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: white
</string>
              </property>
              <property name="text">
               <string>DATOS DE LA MAQUINA</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QFrame" name="buttons_holder_frame">
              <property name="minimumSize">
               <size>
                <width>110</width>
                <height>30</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <widget class="QToolButton" name="minimize_button">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>0</y>
                 <width>21</width>
                 <height>22</height>
                </rect>
               </property>
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../../icons/minimize-window.png</normaloff>../../icons/minimize-window.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
              </widget>
              <widget class="QToolButton" name="restore_button">
               <property name="geometry">
                <rect>
                 <x>50</x>
                 <y>0</y>
                 <width>21</width>
                 <height>22</height>
                </rect>
               </property>
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../../icons/restore-window.png</normaloff>../../icons/restore-window.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
              </widget>
              <widget class="QToolButton" name="maximize_button">
               <property name="geometry">
                <rect>
                 <x>50</x>
                 <y>0</y>
                 <width>21</width>
                 <height>22</height>
                </rect>
               </property>
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../../icons/maximize-window.png</normaloff>../../icons/maximize-window.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
              </widget>
              <widget class="QToolButton" name="close_button">
               <property name="geometry">
                <rect>
                 <x>90</x>
                 <y>0</y>
                 <width>21</width>
                 <height>22</height>
                </rect>
               </property>
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../../icons/close-window.png</normaloff>../../icons/close-window.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="content_frame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="label_3">
            <property name="geometry">
             <rect>
              <x>80</x>
              <y>50</y>
              <width>131</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>NOMBRE MAQUINA</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEdit_3">
            <property name="geometry">
             <rect>
              <x>80</x>
              <y>80</y>
              <width>200</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
border: 1px solid red;</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_4">
            <property name="geometry">
             <rect>
              <x>80</x>
              <y>140</y>
              <width>71</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>SECCION</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEdit_4">
            <property name="geometry">
             <rect>
              <x>80</x>
              <y>170</y>
              <width>200</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
border: 1px solid red;</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_5">
            <property name="geometry">
             <rect>
              <x>330</x>
              <y>240</y>
              <width>49</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>AREA</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEdit_5">
            <property name="geometry">
             <rect>
              <x>250</x>
              <y>270</y>
              <width>200</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
border: 1px solid red;</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_6">
            <property name="geometry">
             <rect>
              <x>420</x>
              <y>50</y>
              <width>131</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>URL DEL MANUAL</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEdit_6">
            <property name="geometry">
             <rect>
              <x>420</x>
              <y>80</y>
              <width>200</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
border: 1px solid red;</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_7">
            <property name="geometry">
             <rect>
              <x>420</x>
              <y>140</y>
              <width>161</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>SELECCIONE UNA IMAGEN</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEdit_7">
            <property name="geometry">
             <rect>
              <x>420</x>
              <y>170</y>
              <width>151</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
border: 1px solid red;</string>
            </property>
           </widget>
           <widget class="QToolButton" name="toolButton">
            <property name="geometry">
             <rect>
              <x>580</x>
              <y>170</y>
              <width>31</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../icons/open-file.png</normaloff>../../icons/open-file.png</iconset>
            </property>
           </widget>
           <widget class="QPushButton" name="add_edit_button">
            <property name="geometry">
             <rect>
              <x>310</x>
              <y>340</y>
              <width>85</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{background-color: rgb(170, 0, 0);
color: white;}
QPushButton::hover{background-color:#ffc13b}</string>
            </property>
            <property name="text">
             <string>text</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../icons/plus.png</normaloff>../../icons/plus.png</iconset>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
